Makefile

```diff
@@ -66,7 +66,9 @@ dirs:
 	./$(AS) -d -i $< -o $@
 
 %.bin: %.mem
-	xxd -r -p $< > $@
+	sed -i -e '$$a1111111111111111111111111111111111111111111111111111111111111111' $^
+	sed -i -e '$$a1111111111111111111111111111111111111111111111111111111111111111' $^
+	perl -ne 'print pack("B64", $$_)' < $^ > $@
```

Comment:
A few questions:

1. What determines the width of this string?
2. Are the "$$" there for matching by the perl command?
3. Why is the rest of the string "a111....1"? What does the "a" mean?
4. Perhaps add a comment above the Make target to explain the termination and why it is needed. It is not obvious from the code in the target.